integreat_cms/api/urls.py

```diff
@@ -219,6 +219,7 @@
 #: The url patterns of this module (see :doc:`django:topics/http/urls`)
 urlpatterns: list[URLPattern] = [
     path("api/v3/regions/", include(region_api_urlpatterns)),
+    path("api/v3/webhook/zammad/", user_chat.zammad_webhook, name="zammad_webhook"),
```

Comment:
Please note: `/api/v3/webhook/...` is a generic new path for webhooks.

Reply:
Could you add some documentation for that at https://github.com/digitalfabrik/integreat-cms/blob/develop/docs/src/api-docs.rst?